### PR TITLE
Validate `revertReason` property on Tx Receipt

### PR DESCRIPTION
**Pull request by ricardolyn**
Tests https://github.com/ConsenSys/quorum/pull/1215

Signed-off-by: Ricardo Silva <ricardo.silva@consensys.net>

### CODE REVIEW

File: networks/typical/main.tf [hunk 1]
```diff
@@ -13,51 +13,69 @@ provider "docker" {
 
 locals {
   number_of_nodes = var.number_of_nodes
-  node_indices    = range(local.number_of_nodes)
+  node_indices = range(local.number_of_nodes)
   more_args = join(" ", [
-    "--allow-insecure-unlock" # since 1.9.7 upgrade
+    "--allow-insecure-unlock",
+    # since 1.9.7 upgrade,
```
Comment (baptiste-b-pegasys) [inline networks/typical/main.tf:19]:
```suggestion
    "--allow-insecure-unlock", # since 1.9.7 upgrade
```
the comment feels not related to the line anymore, is it possible to put like this?

File: networks/typical/main.tf [hunk 2]
```diff
@@ -13,51 +13,69 @@ provider "docker" {
...
+        q2t_start = 49081
+      }
     }
   }
 }
 
 module "network" {
   source = "../_modules/ignite"
 
-  concensus            = module.helper.consensus
```
Comment (baptiste-b-pegasys) [inline networks/typical/main.tf:51]: is this a kind of auto-format that does all these modifications?

**Review by baptiste-b-pegasys — APPROVED** [review]
Just minor comments about terraform formatting

File: networks/typical/main.tf (hunk 2 above)
Comment (dolanor) [reply networks/typical/main.tf:51]: also concensus instead of consensus

Comment (ricardolyn) [reply networks/typical/main.tf:51]: yes, is an auto format. I need to match the format of intellij with terraform fmt. I will push a new branch to fix that.

@dolanor good point. I will fix that typo too.

File: networks/typical/main.tf (hunk 1 above)
Comment (ricardolyn) [reply networks/typical/main.tf:19]: I will fix in a separate branch (used auto merge so as you approved, it merged).

Comment (ricardolyn) [reply networks/typical/main.tf:19]: fixed on https://github.com/ConsenSys/quorum-acceptance-tests/pull/121

File: networks/typical/main.tf (hunk 2 above)
Comment (ricardolyn) [reply networks/typical/main.tf:51]: fixed on https://github.com/ConsenSys/quorum-acceptance-tests/pull/121
